.gitignore

```diff
@@ -1 +1,2 @@
 .*
+/tmp
```

Comment:
What is the correct way? 
My recognition is that "/tmp" matches for only PROJECT_ROOT/tmp directory, and "tmp" matches any directory under the PROJECT_ROOT such as PROJECT_ROOT/test/tmp.